docs/repo/property-pages/property-specification.md

```diff
@@ -441,6 +439,49 @@ internal sealed class MyCommandActionHandler : ILinkActionHandler
 }
 ```
 
+### Focus a property, property page, or property page category
+
+If a hyperlink should, when clicked, set focus to a property page, property page category, or property, the following template may be used.
+
+The editor must specify at least two metadata values:
+
+- `Action`, with value `Focus`
+- `PropertyPage`, with value being the `Name` attribute of the property page to focus.
+
+If you wish to focus a property page category instead of a property page, you must also specify the `PropertyPageCategory` metadata value.
+
+If you wish instead to focus a specific property, you must also specify the `Property` metadata value (**do not** specify `PropertyPageCategory` in this case).
+
+```xml
+<StringProperty Name="MyCommandProperty"
+                DisplayName="Click me to focus a property page category">
+  <StringProperty.DataSource>
+    <DataSource PersistedName="MyCommandProperty"
+                Persistence="ProjectFileWithInterception"
+                HasConfigurationCondition="False" />
+  </StringProperty.DataSource>
+  <StringProperty.ValueEditors>
+    <ValueEditor EditorType="LinkAction">
+      <ValueEditor.Metadata>
+        <NameValuePair Name="Action" Value="Focus" />
+        <NameValuePair Name="PropertyPage" Value="Build" />
+        <NameValuePair Name="PropertyPageCategory" Value="Output" /> <!-- change metadata to Property to focus property, or remove to focus the Build page -->
+      </ValueEditor.Metadata>
+    </ValueEditor>
+  </StringProperty.ValueEditors>
+</StringProperty>
+```
+
+We don't want this property to ever be read from or written to the project file. We intercept these reads and writes by specifying `Persistence="ProjectFileWithInterception"`, and providing the following no-op interceptor. See [Property Value Interception](property-value-interception.md) for more on how and why this works.
```

Comment:
Would it be good to link to the heading on pseudo properties on that page?

Reply:
Yes, it would. I'll add that link